frontend/src/pages/RoomPage.tsx

```diff
@@ -213,6 +216,27 @@ export const RoomPage = () => {
       },
     );
 
+    roomSocket.on(
+      ROOM_EVENTS.SUBMISSION_REJECTED,
+      ({ reason }: SubmissionRejectedPayload) => {
+        enqueueSnackbar(`${reason}`, {
+          variant: "error",
+        });
+        setIsSubmitLoading(false);
+      },
+    );
+
+    roomSocket.on(ROOM_EVENTS.SUBMISSION_ACCEPTED, () => {
+      enqueueSnackbar("Processing submission");
```

Comment:
Should we make the other user's submit button loading too?
```suggestion
      enqueueSnackbar("Processing submission");
      setIsSubmitLoading(true);
```